lib/govuk_components/form_builder.rb

```diff
@@ -1,4 +1,9 @@
 # rubocop:disable Metrics/ClassLength
+# TODO: I've marked this module as not covered by tests because:
```

Comment:
Yep! good call.
Please do remove as much code as possible as you go along with the new gem. We had to implement many tests for all this custom code but going forward it will not be necessary as the new gem is heavily tested, we might just need a few cucumber fixture tests like we did on CAS but that can wait until the migration is more advanced 👍